.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Jakub Melka -->
+<component type="desktop-application">
+  <id>io.github.JakubMelka.Pdf4qt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+  <name>PDF4QT</name>
+  <summary>Open Source PDF Editor</summary>
+  <description>
+    <p>
+      Unleash the power of PDF4QT: Your go-to open source PDF editor powered by the Qt framework. With a robust C++ library, intuitive PDF viewing/editing applications, and a handy command-line tool, PDF4QT streamlines your PDF interactions. Try it today.
```

Comment:
Mentions of `C++` and `Qt framework` are irrelevant to users.

Reply:
Fixed.